docs/devlopment/composer.md

```diff
@@ -0,0 +1,19 @@
+# Composer
+
+A quote from composer website that describes the tool ([https://getcomposer.org](https://getcomposer.org)):
+
+"Composer is a tool for dependency management in PHP. It allows you to declare the libraries your project depends on and it will manage (install/update) them for you."
+
+## Installation
+
+**Note:** Windows users please use a unix shell like `Windows Subsystem for Linux`([https://docs.microsoft.com/en-us/windows/wsl/install-win10](https://docs.microsoft.com/en-us/windows/wsl/install-win10)) or `git bash`([https://git-scm.com/downloads](https://git-scm.com/downloads))
```

Comment:
It's easier to use the installer - in order to add it also to the PATH, etc.
https://getcomposer.org/doc/00-intro.md#using-the-installer or use the manual steps:
https://getcomposer.org/doc/00-intro.md#manual-installation

Reply:
Done